imports/plugins/included/braintree/server/methods/braintreeMethods.js

```diff
@@ -0,0 +1,154 @@
+    Logger.info(refundResult);
+    result = refundResult;
+  } catch (error) {
+    Logger.error(error);
+    result = {
+      saved: false,
+      error: `Cannot Submit Payment: ${error.message}`
+    };
+    Logger.fatal("Braintree call failed, payment was not submitted");
+  }
+
+  return result;
+}
+
+
+/**
+ * paymentCapture
+ * Capture payments from Braintree
+ * https://developers.braintreepayments.com/reference/request/transaction/submit-for-settlement/node
+ * @param {Object} paymentMethod - Object containing everything about the transaction to be settled
+ * @return {Object} results - Object containing the results of the transaction
+ */
+export function paymentCapture(paymentMethod) {
+  check(paymentMethod, PaymentMethod);
+
+  const paymentCaptureDetails = {
+    transactionId: paymentMethod.transactionId,
+    amount: paymentMethod.amount
+  };
+
+  let result;
+
+  try {
+    let refundResult = BraintreeApi.apiCall.captureCharge(paymentCaptureDetails);
+    Logger.info(refundResult);
+    result = refundResult;
+  } catch (error) {
+    Logger.error(error);
+    result = {
+      saved: false,
+      error: `Cannot Capture Payment: ${error.message}`
+    };
+    Logger.fatal("Braintree call failed, payment was not captured");
+  }
+
+  return result;
+}
+
+
+/**
+ * createRefund
+ * Refund BrainTree payment
+ * https://developers.braintreepayments.com/reference/request/transaction/refund/node
+ * @param {Object} paymentMethod - Object containing everything about the transaction to be settled
+ * @param {Number} amount - Amount to be refunded if not the entire amount
+ * @return {Object} results - Object containing the results of the transaction
+ */
+export function createRefund(paymentMethod, amount) {
+  check(paymentMethod, PaymentMethod);
+  check(amount, Number);
+
+  const refundDetails = {
+    transactionId: paymentMethod.transactionId,
+    amount: amount
+  };
+
+  let result;
+
+  try {
+    let refundResult = BraintreeApi.apiCall.createRefund(refundDetails);
+    Logger.info(refundResult);
+    result = refundResult;
+  } catch (error) {
+    Logger.error(error);
+    result = {
+      saved: false,
+      error: `Cannot issue refund: ${error.message}`
+    };
+    Logger.fatal("Braintree call failed, refund was not issued");
+  }
+
+  return result;
+}
+
+
+/**
+ * listRefunds
+ * List all refunds for a transaction
+ * https://developers.braintreepayments.com/reference/request/transaction/find/node
+ * @param {Object} paymentMethod - Object containing everything about the transaction to be settled
+ * @return {Array} results - An array of refund objects for display in admin
+ */
+export function listRefunds(paymentMethod) {
+  check(paymentMethod, Object);
+
+  const refundListDetails = {
+    transactionId: paymentMethod.transactionId
+  };
+
+  let result;
+
+  try {
+    let refundListResult = BraintreeApi.apiCall.listRefunds(refundListDetails);
+    Logger.info(refundListResult);
```

Comment:
They have all been updated to Debug, i think it makes more sense. Especially for listingrefunds, which calls even if no refunds have been issued, so the INFO was showing up blank